metricbeat/etc/beat.full.yml

```diff
@@ -153,10 +153,20 @@ metricbeat.modules:
   #period: 10s
 
   # The host must be passed as PostgreSQL DSN. Example:
-  # postgres://pqgotest:password@localhost:5432?sslmode=disable
+  # postgres://localhost:5432?sslmode=disable
   # The available parameters are documented here:
   # https://godoc.org/github.com/lib/pq#hdr-Connection_String_Parameters
-  #hosts: ["postgres://postgres@localhost:5432"]
+  #
+  # Warning: specifying the user/password in the hosts array is possible
```

Comment:
Not sure if I would add this to the beat.full.yml as not all parts which are in the docs also have to be in the config file. As long as we removed username / password from the example and have username / password below I don't think we need to add this here. It will make it only harder to keep it up-to-date in case of changes.